awx/main/utils/filters.py

```diff
@@ -95,6 +93,22 @@ def filter(self, record):
             return bool(base_name in loggers)
 
 
+class DynamicLevelFilter(Filter):
+
+    def filter(self, record):
+        """Filters out logs that have a level below the threshold defined
+        by the databse setting LOG_AGGREGATOR_LEVEL
+        """
+        # Do not filter loggers in blacklist here, these should be written to file
+        for logger_name in LOGGER_BLACKLIST:
+            if record.name.startswith(logger_name):
+                return True
```

Comment:
This might be too permissive. Before this PR, it defaulted to WARNING, but here, it defaults to not filtering. That's probably okay if some other level filter is in place. While that's true in many cases (very specifically, `awx.conf`), it's not true for some very particular cases.

```
awx/main/utils//reload.py:    logger.debug('Issuing command to {} services, args={}'.format(command, args))
awx/main/utils//handlers.py:        logger.debug('Creating external log handler due to startup or settings change.')
```

I'm about to test this anyway, so seeing if this DEBUG message goes to file logs should tell clearly if this needs to be changed.

If this does need to be changed, then we'll just use the WARNING log level, as opposed to returning true.

Reply:
My theory was right here, I confirmed that I get the message

```
2019-05-22 17:46:02,465 DEBUG    awx.main.utils.handlers Creating external log handler due to startup or settings change.
```

when the level is set to INFO. This will require an additional revision to this PR, which will be coming soon.